blueprint/core/blueprint_ScrollView.h

```diff
@@ -20,7 +20,39 @@ namespace blueprint
         methods delegate to a single child juce::Viewport.
      */
     class ScrollView : public View
+                     , private juce::Timer
     {
+        //==============================================================================
+        class ScrollViewViewport : public juce::Viewport
+        {
+            using OnAreaChangedCallback = std::function<void(const juce::Rectangle<int>&)>;
+        public:
+            ScrollViewViewport() = default;
+
+            void OnAreaChanged(OnAreaChangedCallback callback)
```

Comment:
Major nitpick: lower case for the method name's first letter please! Clang-format will save us soon